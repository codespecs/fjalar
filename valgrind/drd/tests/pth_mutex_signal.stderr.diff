--- pth_mutex_signal.stderr.exp	2022-08-03 11:26:08.247760111 -0700
+++ pth_mutex_signal.stderr.out	2022-08-04 11:53:18.731873751 -0700
@@ -6,10 +6,42 @@
 signalling
 sleeping
 nullHandler running
-unlocking
+Thread 2:
+The impossible happened: mutex is locked simultaneously by two threads: mutex 0x........, recursion count 1, owner 1.
+   at 0x........: pthread_mutex_lock (drd_pthread_intercepts.c:?)
+   by 0x........: contender_start (pth_mutex_signal.c:?)
+   by 0x........: vgDrd_thread_wrapper (drd_pthread_intercepts.c:?)
+   by 0x........: start_thread
+   by 0x........: clone (in /...libc...)
+mutex 0x........ was first observed at:
+   at 0x........: pthread_mutex_init (drd_pthread_intercepts.c:?)
+   by 0x........: main (pth_mutex_signal.c:?)
+
 contender locked mutex
 contender unlocking mutex
 contender unlocked mutex
+unlocking
+Thread 1:
+Mutex not locked by calling thread: mutex 0x........, recursion count 1, owner 2.
+   at 0x........: pthread_mutex_unlock (drd_pthread_intercepts.c:?)
+   by 0x........: main (pth_mutex_signal.c:?)
+mutex 0x........ was first observed at:
+   at 0x........: pthread_mutex_init (drd_pthread_intercepts.c:?)
+   by 0x........: main (pth_mutex_signal.c:?)
+
 joining thread
+Mutex still locked at thread exit: mutex 0x........, recursion count 1, owner 2.
+   at 0x........: pthread_join (drd_pthread_intercepts.c:?)
+   by 0x........: main (pth_mutex_signal.c:?)
+mutex 0x........ was first observed at:
+   at 0x........: pthread_mutex_init (drd_pthread_intercepts.c:?)
+   by 0x........: main (pth_mutex_signal.c:?)
+
+Destroying locked mutex: mutex 0x........, recursion count 1, owner 0.
+   at 0x........: main (pth_mutex_signal.c:?)
+mutex 0x........ was first observed at:
+   at 0x........: pthread_mutex_init (drd_pthread_intercepts.c:?)
+   by 0x........: main (pth_mutex_signal.c:?)
+
 
-ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
+ERROR SUMMARY: 4 errors from 4 contexts (suppressed: 0 from 0)
