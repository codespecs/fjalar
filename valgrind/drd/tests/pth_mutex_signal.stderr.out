
mutex initialized
thread attributes initialized
thread created
sleeping
signalling
sleeping
nullHandler running
Thread 2:
The impossible happened: mutex is locked simultaneously by two threads: mutex 0x........, recursion count 1, owner 1.
   at 0x........: pthread_mutex_lock (drd_pthread_intercepts.c:?)
   by 0x........: contender_start (pth_mutex_signal.c:?)
   by 0x........: vgDrd_thread_wrapper (drd_pthread_intercepts.c:?)
   by 0x........: start_thread
   by 0x........: clone (in /...libc...)
mutex 0x........ was first observed at:
   at 0x........: pthread_mutex_init (drd_pthread_intercepts.c:?)
   by 0x........: main (pth_mutex_signal.c:?)

contender locked mutex
contender unlocking mutex
contender unlocked mutex
unlocking
Thread 1:
Mutex not locked by calling thread: mutex 0x........, recursion count 1, owner 2.
   at 0x........: pthread_mutex_unlock (drd_pthread_intercepts.c:?)
   by 0x........: main (pth_mutex_signal.c:?)
mutex 0x........ was first observed at:
   at 0x........: pthread_mutex_init (drd_pthread_intercepts.c:?)
   by 0x........: main (pth_mutex_signal.c:?)

joining thread
Mutex still locked at thread exit: mutex 0x........, recursion count 1, owner 2.
   at 0x........: pthread_join (drd_pthread_intercepts.c:?)
   by 0x........: main (pth_mutex_signal.c:?)
mutex 0x........ was first observed at:
   at 0x........: pthread_mutex_init (drd_pthread_intercepts.c:?)
   by 0x........: main (pth_mutex_signal.c:?)

Destroying locked mutex: mutex 0x........, recursion count 1, owner 0.
   at 0x........: main (pth_mutex_signal.c:?)
mutex 0x........ was first observed at:
   at 0x........: pthread_mutex_init (drd_pthread_intercepts.c:?)
   by 0x........: main (pth_mutex_signal.c:?)


ERROR SUMMARY: 4 errors from 4 contexts (suppressed: 0 from 0)
